docs/tutorials/basic_networking.md

```diff
@@ -4,7 +4,7 @@
 # Basic Networking Guide for Podman
 
 
-It seems once people master the basics of containers, networking is one of the first
```

Comment:
This feels forced, but fine.

Reply:
How is `conquer`more inclusive? :-)

I am not a native speaker so just ignore my note if you disagree, but the new text is not really clear.  Maybe "learn", "understand"?

Reply:
I think understand is correct, None of us have really mastered or conquered it.

Reply:
Changing to understand, good thought @giuseppe